packages/nouns-contracts/contracts/governance/NounsDAOLogicV2.sol

```diff
@@ -901,6 +964,14 @@ contract NounsDAOLogicV2 is NounsDAOStorageV2, NounsDAOEventsV2 {
         }
     }
 
+    function _refundGas(uint256 startGas) internal {
+        uint256 gasPrice = min(tx.gasprice, block.basefee + MAX_REFUND_PRIORITY_FEE);
+        uint256 gasUsed = startGas - gasleft() + REFUND_TRANSFER_GAS + REFUND_OVERHEAD_GAS;
```

Comment:
@eladmallel maybe worth merging REFUND_TRANSFER_GAS and REFUND_OVERHEAD_GAS to one constant.
in case the compiler doesn't do so already, it's not worth the extra addition operation.
you can specify the breakdown in a comment